python/tskit/trees.py

```diff
@@ -2997,11 +2997,18 @@ def load(file, *, skip_tables=False):
         Please note that with this option set, it is not possible to load data from
         a stream of multiple tree sequences using consecutive calls to
         :meth:`tskit.load`.
+    :param bool skip_reference_sequence: If True, the tree sequence is read
```

Comment:
I wonder if we should say that the ``skip_x`` options require a seekable stream, and they'll fail if you call them on a socket or stdin?

Reply:
yeah, I was wondering about that but wasn't sure how to fit it into the :param structure. should I just throw it in at the top-level before the :param blocks?

Reply:
Throw it in before the :param: blocks yeah, maybe as a ``.. warning::`` or something?